packages/core/botpress/src/web/views/FlowBuilder/common/action.jsx

```diff
@@ -23,7 +23,7 @@ class ActionItem extends Component {
 
   fetchItem = itemId => {
     if (itemId) {
-      this.props.fetchContentItem(itemId, true).then(this.props.refreshFlowsLinks)
+      this.props.fetchContentItem(itemId, { force: true, batched: true }).then(this.props.refreshFlowsLinks)
```

Comment:
Really nice implementation. Funny because that's how I initially thought we should do it!